hack/verify-generated-protobuf.sh

```diff
@@ -35,7 +35,7 @@ trap "cleanup" EXIT SIGINT
 cleanup
 for APIROOT in ${APIROOTS}; do
   mkdir -p "${_tmp}/${APIROOT}"
-  cp -a -T "${KUBE_ROOT}/${APIROOT}" "${_tmp}/${APIROOT}"
+  cp -a "${KUBE_ROOT}/${APIROOT}"/* "${_tmp}/${APIROOT}"
```

Comment:
IMO trailing slash on the target directory changes there nothing and probably this is why it was not included at the start.
If You want - I can add it, but I'm not sure about what would be the purpose of this.